Modules/mathmodule.c

```diff
@@ -2031,6 +2031,87 @@ math_fmod_impl(PyObject *module, double x, double y)
         return PyFloat_FromDouble(r);
 }
 
+/* AC: XXX Todo */
+static PyObject *
+math_dist(PyObject *self, PyObject *args)
+{
+    PyObject *item, *p, *q;
+    double *diffs;
+    double max = 0.0;
+    double csum = 0.0;
+    double x, px, qx, result;
+    Py_ssize_t i, m, n;
+    int found_nan = 0;
+
+    if (!PyArg_ParseTuple(args, "O!O!:dist",
+                          &PyTuple_Type, &p,
+                          &PyTuple_Type, &q)) {
+        return NULL;
+    }
+    m = PyTuple_GET_SIZE(p);
+    n = PyTuple_GET_SIZE(q);
+    if (m != n) {
+        PyErr_SetString(PyExc_ValueError,
+                        "both points must have the same number of dimensions");
+        return NULL;
+
+    }
+    diffs = (double *) PyObject_Malloc(n * sizeof(double));
+    if (diffs == NULL)
+        return NULL;
+    for (i=0 ; i<n ; i++) {
```

Comment:
Always put spaces around assignment, Boolean and comparison operators (PEP7).

Reply:
Historically, we haven't done this inside for-loops.